### ISSUE TITLE
vue ui get error message in locale zh-cn

### ISSUE THREAD

**Issue by dousybox** [vuejs/vue-cli#3410]
### Version
3.4.0

### Environment info
```
  System:
    OS: Windows 7
    CPU: (4) x64 Intel(R) Core(TM) i5-3230M CPU @ 2.60GHz
  Binaries:
    Node: 10.15.0 - D:\Program Files\nodejs\node.EXE
    Yarn: Not Found
    npm: 6.7.0 - D:\Program Files\nodejs\npm.CMD
  npmGlobalPackages:
    @vue/cli: Not Found
```

### Steps to reproduce
1. `npm install -g @vue/cli`
2. `vue ui`
3. system default browser will start automaticlly
4. change browser language to `zh-cn`
5. refresh the `Vue Project Manager`, language will change to zh-cn automaticlly
5. open the develop tool in browser, get the error message: `Uncaught (in promise) Error: Cannot find module './zh'`

### What is expected?
no error message in browser console,
and pages can work fast.

### What is actually happening?
error message `Uncaught (in promise) Error: Cannot find module './zh'` in browser console;
and the pages are woking very slowlly.

---
Runing `vue ui` has encured some error when the page switch to zh-cn language automaticlly.
![image](https://user-images.githubusercontent.com/33720723/52172068-00ea0500-27a3-11e9-88c2-68efbcb86c37.png)

Is there any way to the the default locale language for the `Vue Project Manager`?

**Comment by arthurdenner** [comment]
I investigated this issue and noticed that because of some "inconsistencies" between browser language, UI's locales and date-fns, this problem happens with Chinese and Brazilian Portuguese (and maybe others).

For Chinese, the browser gives us `["zh", "zh", "zh"]` as language codes, the UI has `zh.json`, but `date-fns` has only `zh_cn`;

For Brazilian Portuguese, the browser gives us `["pt-BR", "pt-br", "pt"]` as language codes, the UI has `pt_BR.json` (so it quits the `for` loop [here](https://github.com/vuejs/vue-cli/blob/dev/packages/%40vue/cli-ui/src/i18n.js#L45)), but `date-fns` has only `pt`;

@sodatea, any idea on how to proceed about this?

**Comment by sodatea** [comment]
Fixed in #3966
